.circleci/config.yml

```diff
@@ -99,11 +114,12 @@ jobs:
             curl $CBIOPORTAL_URL > /dev/null && \
             sleep 20s && \
             curl http://localhost:3000 > /dev/null && \
-            sleep 20s && \
+            sleep 1s && \
```

Comment:
no need to sleep this long now since we don't have to wait for frontend artifacts to be build as is the case with `npm run start`